code/datums/antagonists/wizard.dm

```diff
@@ -148,7 +148,7 @@
 	var/wizard_name_second = pick(GLOB.wizard_second)
 	var/randomname = "[wizard_name_first] [wizard_name_second]"
 	var/mob/living/wiz_mob = owner.current
-	var/newname = copytext(sanitize(input(wiz_mob, "You are the [name]. Would you like to change your name to something else?", "Name change", randomname) as null|text),1,MAX_NAME_LEN)
+	var/newname = copytext(sanitize(input(wiz_mob, "You are [src]. Would you like to change your name to something else?", "Name change", randomname) as null|text),1,MAX_NAME_LEN)
```

Comment:
I believe this will output as You are Space Wizard.

Reply:
Which, if "Space" and "Wizard" are indeed the chosen first and last names, is grammatically correct.

Reply:
Yes